src/sage_model.js

```diff
@@ -118,16 +118,8 @@ const model = function(name, schema, sage) {
             }
             break;
 
-          case 'number':
```

Comment:
Why are these checks not needed?

Reply:
This isn't so much of a check as it is formatting any number or numeric string into an integer which is problematic because Oracle Number types are not just limited to integers. There's still a validation function which verifies you're using a number. There's a use for this type of functionality, but it's better off in a user-defined converter function (functionality we would need to add).